client/src/dashboard/RewritingRuleModal.js

```diff
@@ -182,11 +183,39 @@ const RewritingRuleModal = NiceModal.create(({user_id, rule=null, query=null}) =
     });
   };
 
-  function findDistanceToSpaces(inputString, currentPosition) {
+  const onBeautifyQuery = () => {
+    // Beautify original query based on selected sql languages(mysql or pgsql)
+    const sqlLanguage = (queryLanguage === 'pgsql') ? "postgresql" : queryLanguage;
```

Comment:
Can we make the initial list of options compatible with the query languages defined in sql-formater? Right now it seems only the `pgsql` is inconsistent with `postgresql`.

Reply:
Regarding PostgreSQL, the Ace editor library (which I use for syntax highlighting and auto-completion) only accepts "pgsql" as the keyword, while the SQL formatter library only accepts "postgresql" as the keyword. I attempted to merge them into one but it did not work out so far :(